### DISCUSSION TITLE
v19.5.0-rc3

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3169] [Releases]
## Highlights
This is a big release, that's why we decided to move away from the 19.4.x versioning and go to 19.5.x. Since a lot of changes could have some impact on current installations we decided to release some "rc" versions before the official 19.5.0. Tests are more than welcome now but be extra careful with production environment.

- What's most important is the removal of all the 3rd party libraries (phpseclib, mcrypt_compat, Cm_RedisSession, Cm_Cache_Backend_Redis and Pelago_Emogrifier and Zend Framework) form our repository, they are now imported via composer. This was an important step to clean up and modernise our code.
- PHP 7.4 is now the minimum required version and 8.2 is now supported
- M1 legacy themes have been moved to an external repository since it's old (and mostly unused) code.
- Support for Google Analytics 4 was added

Don't worry though, if you've always installed OpenMage extracting the zip file, starting from this release you'll find a new zip file attached to the release itself, we build this zip adding all of the old 3rd party libraries so that you will not have to migrate to composer or use composer at all.

## Changelog
* Fix exception logged when no Category image is uploaded by @elidrissidev in https://github.com/OpenMage/magento-lts/pull/3086
* Added frontend_type color by @fballiano in https://github.com/OpenMage/magento-lts/pull/2945
* PHP 7.4 is now the minimum required version by @fballiano in https://github.com/OpenMage/magento-lts/pull/3091
* Fixed ESI parsing error with turpentine by @luigifab in https://github.com/OpenMage/magento-lts/pull/2987
* Added new options to"number of record" in grid by @loekvangool in https://github.com/OpenMage/magento-lts/pull/3096
* Fixed allowed PHP version in composer.json by @fballiano in https://github.com/OpenMage/magento-lts/pull/3097
* Some PHPStan fixes by @fballiano in https://github.com/OpenMage/magento-lts/pull/3031
* Added info about Mage_Poll removal to README by @fballiano in https://github.com/OpenMage/magento-lts/pull/3101
* Deprecation errors are not suppressed anymore by @fballiano in https://github.com/OpenMage/magento-lts/pull/3102
* Bump phpstan/phpstan from 1.10.6 to 1.10.7 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3104
* Bump PHPCSFixer to 3.15.1 + new fixes required by @fballiano in https://github.com/OpenMage/magento-lts/pull/3106
* Fixed PHP_OS check in cron.php by @elidrissidev in https://github.com/OpenMage/magento-lts/pull/3113
* Bump dev-meta-package to 1.0.2 by @fballiano in https://github.com/OpenMage/magento-lts/pull/3111
* Removed commented code by @fballiano in https://github.com/OpenMage/magento-lts/pull/3100
* Fixed problem with associated_products not visible in backend by @fballiano in https://github.com/OpenMage/magento-lts/pull/3093
* Bump phpstan/phpstan from 1.10.7 to 1.10.8 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3117
* Bump colinmollenhour/magento-redis-session from 3.0.2 to 3.1.1 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3116
* Fixed Illegal offset type in Mage_Tag by @m-overlund in https://github.com/OpenMage/magento-lts/pull/3118
* PHPStan: some fixes to lib/ files by @fballiano in https://github.com/OpenMage/magento-lts/pull/3099
* Disabled and disallowed SWF file extension by @fballiano in https://github.com/OpenMage/magento-lts/pull/3108
* Prevented Configurable Swatches assets from loading while unneeded by @loekvangool in https://github.com/OpenMage/magento-lts/pull/2999
* [PHP 8.2] Fixed deprecated dynamic property creation by @elidrissidev in https://github.com/OpenMage/magento-lts/pull/3094
* RWD theme: Focus on the search field when clicking on the search button by @fballiano in https://github.com/OpenMage/magento-lts/pull/3095
* Fixed monthly report dates (date was 1970) by @sreichel in https://github.com/OpenMage/magento-lts/pull/3126
* Bump phpunit/phpunit from 9.6.5 to 9.6.6 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3131
* Bump friendsofphp/php-cs-fixer from 3.15.1 to 3.16.0 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3132
* Bump phpstan/phpstan from 1.10.8 to 1.10.10 by @dependabot in https://github.com/OpenMage/magento-lts/pull/3133
* Updated license info in copyright docblocks by @fballiano in https://github.com/OpenMage/magento-lts/pull/3120

**Full Changelog**: https://github.com/OpenMage/magento-lts/compare/v19.5.0-rc2...v19.5.0-rc3

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v19.5.0-rc3'>v19.5.0-rc3</a>.</em>

**Comment by djixas** [comment]
Is next build RC4 or final?

**Reply by fballiano** [reply]
I think there will be some more RCs before a final :-)
